packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -149,3 +149,17 @@ export function processAdditionalValidOrigins(validMessageOrigins: string[]): vo
   });
   GlobalVars.additionalValidOrigins = combinedOriginUrls;
 }
+
+/**
+ * @hidden
+ * Check if apiVersionTag developer sends follows the pattern starting with a lowercase 'v', then
+ * followed by one or more digits, then concated with underscore and some characters to indicate api name.
```

Comment:
```suggestion
 * followed by one or more digits, then concatenated with underscore and some characters to indicate api name.
```

Reply:
marked as will be updated in next commit.